packages/backend-seed/API.md

```diff
@@ -0,0 +1,12 @@
+## API Report File for "@aws-amplify/backend-seed"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export const GetSeedSecret: (secretName: string) => Promise<string>;
```

Comment:
I'd just call it `getSecret` .

Other option we could consider is 

```
type SeedSecretClient {
  getSecret: () => Promise<string>;
}

// factory returns implementation
getSeedSecretClient = () => SeedSecretClient
```

We should think about packaging of this because APIs to deal with secrets will grow over time.
We'll also have `AuthSeedClient` or so or set of auth related APIs. it would be good to group secrets and auth.

An alternative would be to develop namespaces:
`@aws-amplify/backend-seed/secret` and `@aws-amplify/backend-seed/auth` .

We should capture these alternatives in API review.

Reply:
I initially called it `getSecret`, but I got some errors since it conflicted with `getSecret` from the `backend-secret` package -- I'll probably experiment with it again

Reply:
This might be helpful syntax to resolve that problem https://github.com/aws-amplify/amplify-backend/blob/b574bce26ec2d55a4391aeceecc1d4957e79fba3/packages/backend-output-schemas/src/auth/index.ts#L2 .

or `import * as backendSecret from 'backend-secret'` and then `backendSecret.getSecret` .